### DISCUSSION TITLE
김일태

### DISCUSSION THREAD

**Discussion by z1nun** [z1nun/ASTERISK-#12] [독서일지]
# 
![free-icon-reading-3749934](https://user-images.githubusercontent.com/106229016/194359382-e93bdf18-fd7f-4888-9284-778dd549ffd3.png)

## 클린 코드 (Clean Code)

- 2022.10.06 p1 - p20

**Comment by iltae** [comment]
**1. 깨끗한 코드**

- 코드는 요구사항을 상세히 표현하는 수단
- 중복을 피하라. 한 기능만 수행하라. 제대로 표현하라. 작게 추상화하라
- 보이스카우트 규칙 (캠프장은 처음 왔을 때보다 더 깨끗하게 해놓고 떠나라)
